examples/record-iteration/record_iteration.bal

```diff
@@ -0,0 +1,57 @@
+import ballerina/io;
+
+type Grades record {
+    int maths;
+    int physics;
+    int chemistry;
+    !...
+};
+
+public function main() {
+    // This creates a `Grades` record.
+    Grades grades = {maths: 80, physics: 75, chemistry: 65};
+
+    // The foreach variable is a 2-tuple which consists of the field name and the value of the field.
```

Comment:
```suggestion
    // The `foreach` variable is a 2-tuple, which consists of the field name and the value of the field.
```

Shall we explain the 'n-tuple' concept in brief?

You'd normally have a comma before 'which' in this context.

Reply:
Do we have to? That's a usual naming convention used w.r.t. tuples and not related to Ballerina. Anyone familiar with the tuple concept should know what is meant by an 'n-tuple'. While I haven't so far seen that used in Ballerina, used it here to easily (and formally) say that the foreach variable is a tuple with two elements.

Reply:
Ok noted. If it's understood by anyone attempting to learn the language, this is fine.